### DISCUSSION TITLE
A question about the errors

### DISCUSSION THREAD

**Discussion by bearpawmaxim** [syssi/esphome-jk-bms#573] [Q&A] [answered]
Hello @syssi! I'm that guy, that tries to emulate the PACE BMS with the jkbms [here](https://github.com/syssi/esphome-pace-bms/discussions/42)  :-)

I have a question about the errors value. In terms of a PACE BMS, when we see the error, is it the warning about the protection is going to kick in or it is the the protection that is already active?

**Comment by syssi** [comment, marked as answer]
If you see an error the issue already happend and the BMS is waiting for recovery.

**Reply by bearpawmaxim** [reply]
Thank you!

Does this means that if, for example, I've got the charging overtemperature protection, the BMS stopped the charging?

And, jkbms doesn't provides the warnings?
Even if yes, they are computable..

**Reply by syssi** [reply]
Just cool down the temperature sensor using some ice and give it a try. ;-) If the charging over/undertemperature kicks in the `error` sensor will indicate the type of protection and the `charging` binary sensor turns off while the `charging` switch stays on.
